tests/conftest.py

```diff
@@ -232,6 +232,19 @@ def postgres_db(request: pytest.FixtureRequest):
     yield request.getfixturevalue(fixture)
 
 
+@pytest.fixture
+async def postgres_db_debug(postgres_db, database_name) -> abc.AsyncIterator[None]:
```

Comment:
I used this to interact with the database in the context of the test case I was debugging. I think it could be useful to keep it and refer to it from our internal docs. Wdyt?

Reply:
@arnaudsjs rerequesting review for this change. I'm creating a follow-up ticket for the performance fix right now.